docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+
+## 3. Connect your receiver to your flight controller
+
+For the 1 watt MatekSys [mLRS receivers](https://www.mateksys.com/?portfolio=mr24-30#tab-id-2):
+- connect the receiver Rx1 to a FC UART Tx.
+- connect the receiver Tx1 to the corresponding FC UART Rx.
+- Connect the receiver V pin to a 5 volt output on your FC.
+- Connect the receiver G pin to a ground pad on your FC.
+- If you will use CRSF for RC channels (recommended), connect the receiver TX2 to a different UART Rx on your FC.
+
+For most ELRS receivers, the [connections](ELRS_RECEIVERS.md#connections) are similar to those above except CRSF is not available.
+
+Connect your ground station to your ArduPilot flight controller via USB and configure the serial port (x) you wired to your receiver serial port.  Don't forget to press "write parameters" if your ground station requires this step.
+- Set SERIALx\_PROTOCOL to "MAVLink2" (2).
+- Set SERIALx\_BAUD to "57600" (57) (default mLRS Rx Ser Baudrate).
+
+If you will use CRSF or SBUS for RC channels, use your ground station to configure the serial port (x) you wired to your receiver RC output.
+- Set SERIALx_PROTOCOL to "RCIN" (23).
+- Set RC_PROTOCOLS to 536.
+- Set RSSI_TYPE to 3.
+
+## 4. Install firmware if required
+
+If both your Rx and Tx modules came pre-installed with mLRS firmware as with the MatekSys mLRS modules, you can use the existing firmware unless you need a bug fix or feature which was added in a newer firmware version.
+
+Carefully install your Tx module into the bay of your EdgeTX or OpenTX radio.  Some Tx modules can be fussy about pin alignment;  don't use excessive force or you may break something.
+
+For the R9M Tx module, it is recommended to [Flash via EdgeTx/OpenTx](FRSKY_R9.md#flash-the-r9m-module-with-elrs-bootloader) which is a 2 step process:
+
+- First, [Flash the ELRS Bootloader](FRSKY_R9.md#flash-the-elrs-bootloader).
+- Next, use your EdgeTX/OpenTX radio to [Flash the mLRS Firmware](FRSKY_R9.md#flashupdate-the-mlrs-firmware).
+
+To flash ELRS receivers, you can use esptool or a Chrome or Chromium based browser as instructed [here](ELRS_RECEIVERS.md#flashing)
+
+[Flashing firmware](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-3) to the MatekSys mLRS Tx modules is especially easy.
+
+Once the firmware is installed, you can confirm operation by observing the LED.
+- When either a receiver or Tx module is powered on by itself, a LED (usually red) should flash at 2 Hz (twice per second, on for 1/4 second then off for 1/4 second repeatedly).
+- When the Tx module and receiver are both powered on and bound, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
```

Comment:
ok, here we have to be carefull ... from the discord it seems that many users easily confuse binding and connecting, and I guess we don't want to add to it
maybe "... are both powered on and are bound, they will connect and a LED ..."